package.json

```diff
@@ -53,15 +58,19 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.12.0",
-    "@typescript-eslint/parser": "^5.12.0",
-    "eslint": "^8.9.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.28.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
-    "prettier": "^2.5.1"
+    "@types/react-router-dom": "~5.3.3",
+    "@typescript-eslint/eslint-plugin": "~5.12.0",
+    "@typescript-eslint/parser": "~5.12.0",
```

Comment:
with husky you need to use lint-staged please read the documentation and fix appropriately